src/scripts/main.js

```diff
@@ -1 +1,12 @@
 'use strict';
+
+const frame = document.getElementById('largeImg');
+const thumbs = document.getElementById('thumbs');
+
+thumbs.addEventListener('click', (ev) => {
+  ev.preventDefault();
+
+  const item = ev.target;
```

Comment:
It's better to check if the clicked element is an image before changing the source of the main image.